core/iwasm/aot/aot_intrinsic.c

```diff
@@ -514,6 +542,15 @@ add_intrinsic_capability(AOTCompContext *comp_ctx, uint64 flag)
     }
 }
 
+static void
+add_i64_common_intrinsics(AOTCompContext *comp_ctx)
```

Comment:
who is going to call this function?

Reply:
I guess all 32-bit platform will need it, but only riscv32 verified now.